### PR TITLE
feat: support new sepolia testnet [APE-1487]

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did

fixes: #

### Checklist

- [x] Passes all linting checks (pre-commit and CI jobs)
- [x] New test cases have been added and are passing
- [x] Documentation has been updated
- [x] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

**Comment by fubuloubu** [comment]
related: https://github.com/ApeWorX/ape-etherscan/pull/105

**Comment by fubuloubu** [comment]
related: https://github.com/ApeWorX/ape-infura/pull/66

File: ape_optimism/ecosystem.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
     # chain_id, network_id
     "mainnet": (10, 10),
     "goerli": (420, 420),
+    "sepolia": (11155420, 11155420),
```
[the remark this answers is not in the document]
Comment (fubuloubu) [reply ape_optimism/ecosystem.py:21]: This is the chain ID of the optimism l2 chain for sepolia, not sepolia itself

[the remark this answers is not in the document]
Comment (antazoey) [reply ape_optimism/ecosystem.py:21]: 11155111 is for Ethereum's Sepolia, right? 11155420 is for Optimism's

**Review by antazoey — APPROVED** [review]

**Pull request merged**